test/integration/stats_integration_test.cc

```diff
@@ -265,11 +265,12 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // 2019/10/17  8484     43340       44000   stats: add unit support to histogram
   // 2019/11/01  8859     43563       44000   build: switch to libc++ by default
   // 2019/11/15  9040     43371       44000   build: update protobuf to 3.10.1
-  // 2019/11/15  9040     43403       44000   upstream: track whether cluster is local
+  // 2019/11/15  9031     43403       44000   upstream: track whether cluster is local
```

Comment:
I think you have several merge issues in this file.

Reply:
The 9040->9031 is intentional, the PR did not update this line correctly.  #9040 was the build commit listed above it, #9031 is the one mentioned in the description.

Will fix the 4400->43500 mistake, looks like that value was changed sometime after the original commit.